CPP17.md

```diff
@@ -17,6 +17,7 @@ C++17 includes the following new language features:
 - [constexpr if](#constexpr-if)
 - [utf-8 character literals](#utf-8-character-literals)
 - [direct-list-initialization of enums](#direct-list-initialization-of-enums)
+- [\[\[fallthrough\]\], \[\[nodiscard\]\] and \[\[maybe_unused\]\] attributes](#standard-attributes)
```

Comment:
Even though there are just 3 of them, change this to be `[New standard attributes](#new-standard-attributes)`. Considering C++20 introduces a few more than just 3 attributes, I figure it's easier to do it this way and maintain consistency.

Reply:
I changed it, just keep in mind that when C++20 introduces, we can't use "new standard" again because it won't disambiguate in the `README.md` file